### ISSUE TITLE
Implement Prettier

### ISSUE THREAD

**Issue by JasonEtco** [probot/linter#11]
Hey folks, I heard about this [on your amazing podcast spot](https://changelog.com/podcast/264) and thought about it a little. Have y'all seen [Prettier](https://github.com/prettier/prettier)? I think we could make use of that here. If you're not interested, I'd love to build it out as a separate app thats entirely prettier-based but I think it could just be used here.

**Comment by tunnckoCore** [comment]
Yup. I recently suggested creating prettier-bot https://github.com/prettier/prettier/issues/3262, but the response was quite good - StyleJS. It seems good too, but i feel it does too much and also don't works well with eslint, probably because is not using `prettier-eslint`, but it is `must have`.

I am interested in starting work here too. Few things:
- respect all eslint's config files *
- respect all prettier config files *
- use `prettier-eslint` by default, or some sort of auto-detection - to use it only if eslint config is found in repo, otherwise use only `prettier`

\* - resolver similar to `cosmiconfig` should be implemented that works with github api. It's not so hard btw, following the code of [pretty-config](https://github.com/archieved/pretty-config/blob/master/src/index.mjs) which is a lot smaller and better. I even sit for 2 minutes with cup of coffee and visualized all the things in my head. ProTip: use `safe-eval`. 

I believe the resolver should be implemented in the core.
